### DISCUSSION TITLE
Discovery issues

### DISCUSSION THREAD

**Discussion by ghost** [plasticrake/homebridge-tplink-smarthome#268] [Q&A]
Hello everyone,

I've recently upgraded my Homebridge unit and for some reason I cannot get my TP-Link devices to be discovered.  I have used the UX to load and configure the plugin and UDP is setup so I'm at a loss.

Any help would be greatly appriciated.

Thank you.  Jeff

**Comment by anshukapoor** [comment]
I am new to Homebridge. I installed Homebridge on Raspberry Pi 4 using Raspberry Pi installer and then installed this plugging using the UI. I have configured the broadcast address to match my subnet. I did not change any other setting. I have KP115 device setup, but it is not show up in Homebridge.

**Reply by anshukapoor** [reply]
It finally worked for me after changing the broadcast address to 255.255.255.255.
